lib/google/protobuf.ex

```diff
@@ -163,15 +163,22 @@ defmodule Google.Protobuf do
 
         iex> from_duration(Duration.new!(second: 1, microsecond: {500_000, 6}))
         %Google.Protobuf.Duration{seconds: 1, nanos: 500_000_000}
+
+        iex> from_duration(Duration.new!(hour: 1, minute: 2, microsecond: {500_000, 6}))
+        %Google.Protobuf.Duration{seconds: 3720, nanos: 500_000_000}
+
+        iex> from_duration(Duration.new!(minute: 2, microsecond: {6_500_000, 6}))
+        %Google.Protobuf.Duration{seconds: 126, nanos: 500_000_000}
```

Comment:
These tests would fail on the previous impl

Reply:
yeah... I was waiting for your code review before merging my implementation 😭